jbot/src/main/java/me/ramswaroop/jbot/core/slack/Bot.java

```diff
@@ -187,6 +187,12 @@ protected final void reply(WebSocketSession session, Event event, Message reply)
     }
 
     protected final void reply(WebSocketSession session, Event event, String text) {
+        reply(session, event, text, false);
+    }
+
+    protected final void reply(WebSocketSession session, Event event, String text, boolean mrkdwn) {
+        Message message = new Message(text);
+        message.setMrkdwn(mrkdwn);
         reply(session, event, new Message(text));
```

Comment:
```suggestion
        reply(session, event, message);
```
Should it be passing in the message you build to the reply method?

Reply:
Yes, as you have built the message in the previous line, you can directly pass it to reply() method. Also, can you please share me the link to the Slack API documentation which shows the `mrkdwn` property?

Reply:
i saw mrkdwn property from this url :) 
https://api.slack.com/docs/message-formatting